src/dparse/parser.d

```diff
@@ -188,6 +188,24 @@ class Parser
         return attachCommentFromSemicolon(node, startIndex);
     }
 
+    /**
+     * Parses an AliasAssign.
+     *
+     * $(GRAMMAR $(RULEDEF aliasAssign):
+     *       $(RULE aliasInitializer)
+     */
+    AliasAssign parseAliasAssign()
+    {
+        mixin(traceEnterAndExit!(__FUNCTION__));
+        auto startIndex = index;
+        auto node = allocator.make!AliasAssign;
+        node.initializer = parseAliasInitializer();
```

Comment:
AliasInitializer allows adding template parameters, are these really supported by alias assign? (as this is a public function that may be called freely - and for good code style)

Reply:
Nope. I can implement it properly, but I thought I would just reuse as much code as possible to make the diff smaller.